### ISSUE TITLE
请问推理输入的sp0， sp1是什么意思？

### ISSUE THREAD

**Issue by JohnHerry** [netease-youdao/EmotiVoice#35]
请问推理输入的音素序列中 sp0， sp1 是什么？是停顿标记吗？在推理时是怎么得到的？

**Comment by syq163** [comment]
Yes, the 'sp?' symbols indicate a short pause. A TTS frontend can be utilized to predict and adjust the pause duration. Additionally, you have the flexibility to modify it according to your requirements.
